js-import.el

```diff
@@ -52,7 +52,7 @@
 
 (defun js-import-is-js-file (filename)
   "Check if FILENAME ends with either .js or .jsx."
```

Comment:
great improvement! could you also update the comment to reflect the changes in the code? Thanks :)